### DISCUSSION TITLE
部署在云主机，开启服务器转发怎么能生效

### DISCUSSION THREAD

**Discussion by wangqiang1988** [xfgryujk/blivechat#185] [提问] [answered]
我想通过服务器转发改一下services/chat.py中的显示消息内容，怎么能用云主机的同时使用服务器转发呢

**Comment by xfgryujk** [comment]
前端启用通过服务器转发就行了

**Reply by wangqiang1988** [reply]
前端启用通过服务器转发后，无法收到弹幕。另外tornado_xheaders 改成了true，加载器禁用了，跟这两个有关系吗？

**Reply by xfgryujk** [reply, marked as answer]
> 前端启用通过服务器转发后，无法收到弹幕。另外tornado_xheaders 改成了true，加载器禁用了，跟这两个有关系吗？

没关系，你自己F12看什么问题吧
